lms/templates/verify_student/photo_reverification.html

```diff
@@ -62,7 +62,7 @@ <h3 class="title">${_("Error submitting your images")}</h3>
         <div class="message">
           <h3 class="title">${_("Please Resubmit Your Verification Information")}</h3>
           <div class="copy">
-            <p>${_("There was an error with your previous verification. In order proceed in the verified certificate of achievement track of your current courses, please complete the following steps.")}</p>
+            <p>${_("In order proceed in the verified certificate of achievement track of your current courses, please complete the following steps.")}</p>
```

Comment:
FYI, the heading in line 63 needs a change too. I've included the heading and accompanying text below. (If it's too long, feel free to delete "If you do not verify your identity, you can still receive an honor code certificate for the course.") I've indicated that "before the course verification deadline" should be in bold text, if possible.

Verify your identity

To verify your identity and continue as a verified student in this course, complete the following steps
**before the course verification deadline**. If you do not verify your identity, you can still receive an honor code certificate for the course.